common/lib/xmodule/xmodule/tests/test_xblock_wrappers.py

```diff
@@ -290,6 +290,7 @@ def test_leaf_node(self, cls_and_fields):
         # pylint: disable=no-member
         descriptor.runtime.id_reader.get_definition_id = Mock(return_value='a')
         descriptor.runtime.modulestore = modulestore
+        descriptor._xmodule.graded = 'False'
```

Comment:
Again, I'm concerned about just sticking this in. Why is this necessary? Is this indicative that the code should be more robust to whether or not 'graded' exists?